lib/tessel/deploy.js

```diff
@@ -54,6 +54,26 @@ Tessel.prototype.deployScript = function(opts) {
   });
 };
 
+Tessel.prototype.restartScript = function(opts) {
+  var self = this;
+  var isPush = opts.type === 'flash';
+  var filepath = isPush ? PUSH_PATH : RUN_PATH;
+
+  return new Promise(function(resolve, reject) {
+    actions.findProject(opts.entryPoint).then(function(project) {
```

Comment:
That's not what I'm seeing. I get the "module doesn't exist" failure. 

> I think we should explicitly check that code exists in that remote directory prior to deployment and throw an error if it doesn't.

We can get this _for free_ already, we just need to catch the error and reject with the message accordingly.